invoicer_project/clients/views.py

```diff
@@ -33,17 +32,15 @@ def create(self, request):
     def list(self, request):
         user = get_user_from_jwt(request.headers)
         clients = Client.objects.filter(user_id=user['user_id'])
-        clients_json = ClientSerializer(clients, many=True)
         if len(clients) == 0:
             return JsonResponse(
-                {"first_name": user['first_name'], "last_name": user['last_name'], 'message': 'Not found'}, status=200)
+                {"first_name": user['first_name'], "last_name": user['last_name'], 'message': 'Not found'}, status=404)
+        clients_json = ClientSerializer(clients, many=True)
         data = {"first_name": user['first_name'], "last_name": user['last_name'], 'content': clients_json.data}
```

Comment:
Do you think attributes `first_name` and `last_name` are important in response? I guess user is known on UI side because it store on localStorage.

Reply:
Done